test/fixtures/uphold_connection_for_channels.yml

```diff
@@ -0,0 +1,10 @@
+default: &default
```

Comment:
I think we should be renaming this, but it somehow works? I thought there would be a naming conflict where we use `default` for a publisher (even though the proper name should be `default_publisher` and we'd name this `default_uphold_connection_for_channel`

Reply:
https://github.com/brave-intl/publishers/pull/2150/commits/bbe5ca8e155b51399bc56580afe3ee12375fc1d7